webapp/app/assessment/components/section/assessmentSectionView/assessmentSectionView.js

```diff
@@ -16,6 +16,8 @@ import { fetchLastSectionUpdateTimestamp, resetSectionUpdateTimestamp } from '@w
 import AssessmentSection from './assessmentSection'
 import useSectionTables from './useSectionTables'
 
+import { documentScrollTo } from '../../../../../utils/domUtils'
```

Comment:
can we import with @?

Reply:
my IDE is not recognising them anymore. I would prefer we switch to normal import